html_css_exercise/femc_level1_huddleLandingPage/css/index.css

```diff
@@ -0,0 +1,144 @@
+:root {
+    --background-color-1: #674bb1;
+    font-size: 62.5%;
+}
+
+html,
+body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    overflow: hidden;
+}
+
+body {
+    background-image: url(../img/bg-desktop.svg);
+    background-color: var(--background-color-1);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+a {
+    cursor: pointer;
+}
+
+/****** Header Styling *********/
+
+.header-main {
+    width: 100%;
+    height: 65px;
+    margin: 0 0 40px;
+    padding: 0 40px;
+}
+
+/****** Main Section Styling *********/
+
+.container-main {
+    width: 100%;
+    padding: 0 40px;
```

Comment:
The `rem` unit is more often used for padding and margin than `px`. This is because `rem` scales with browser zoom (aka root text size), as I mentioned elsewhere. So your proportions of padding/text size will be preserved. This is not a hard fast rule however, and you may elect to use other units when applicable.

Reply:
updated, leaving px for heights, widths, and borders